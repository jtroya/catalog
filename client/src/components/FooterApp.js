import React from 'react';
import { Footer } from 'react-materialize';

function FooterApp() {
  return (
      <Footer className="footer_app" copyrights="2018 Copyright Text" />
  );
}

export default FooterApp;
