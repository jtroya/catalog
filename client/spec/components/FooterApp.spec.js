import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import FooterApp from '../../src/components/FooterApp';

describe('FooterApp', () => {
  it('should render the FooterApp', () => {
    const wrapper = shallow(<FooterApp />);
    expect(wrapper.find('Footer')).toBeTruthy();
  });

  it('matches the snapshot', ()=> {
    const tree = shallow(<FooterApp />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
