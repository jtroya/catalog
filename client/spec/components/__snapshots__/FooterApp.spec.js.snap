// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterApp matches the snapshot 1`] = `
<Footer
  className="footer_app"
  copyrights="2018 Copyright Text"
/>
`;
